### ISSUE TITLE
Requiring external module babel-register

### ISSUE THREAD

**Issue by moinhanif** [kevinchappell/formBuilder#443]
I'm using window 10, I have downloaded project from git as zip, and run 
npm install
npm start
```
D:\JqueryFormbuilder\formBuilder-master>npm start

> formBuilder@2.1.2 start D:\JqueryFormbuilder\formBuilder-master
> node ./node_modules/gulp/bin/gulp.js
[10:24:49] Requiring external module babel-register
[10:24:53] Using gulpfile D:\JqueryFormbuilder\formBuilder-master\gulpfile.babel.js
[10:24:53] Starting 'devJS'...
[10:24:54] Finished 'devJS' after 810 ms
[10:24:54] Starting 'devCss'...
[10:24:55] Finished 'devCss' after 1.85 s
[10:24:55] Starting 'watch'...
[10:24:56] Finished 'watch' after 264 ms
[10:24:56] Starting 'serve'...
[10:24:56] Finished 'serve' after 87 ms
[10:24:56] Starting 'default'...
[10:24:56] Finished 'default' after 1.6 μs
[BS] Access URLs:
 ------------------------------------
       Local: http://localhost:3000
    External: http://10.11.11.26:3000
 ------------------------------------
          UI: http://localhost:3001
 UI External: http://10.11.11.26:3001
 ------------------------------------
[BS] Serving files from: ./demo
Terminate batch job (Y/N)?
```

when so when i run glup -v
```
D:\JqueryFormbuilder\formBuilder-master>gulp -v
[10:40:12] Requiring external module babel-register
[10:40:12] CLI version 3.9.1
[10:40:12] Local version 3.9.1
```

screenshot of the localhost
![scrrenshtot](https://cloud.githubusercontent.com/assets/25588668/25734729/689e7940-317f-11e7-9d0a-554a77465313.PNG)

**Comment by sunyinghao5438** [comment]
+1

**Comment by sunyinghao5438** [comment]
Object doesn't support this action

**Comment by kevinchappell** [comment]
resolved in v2.2
